### PR TITLE
AC-623: Name and other details are not showing when searching patients

### PR DESCRIPTION
**Pull request by vansha10**
## Description of what I changed

In download patients, only the Patient ID was showing, now the name is also showing.

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-623

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

## Screenshot:
Screenshot_20190630-155431

### CODE REVIEW

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/lastviewedpatients/LastViewedPatientRecyclerViewAdapter.java [hunk 1]
```diff
@@ -121,6 +121,13 @@ public void onBindViewHolder(@NonNull RecyclerView.ViewHolder holder, int positi
             }
             if (null != patient.getName()) {
                 ((PatientViewHolder) holder).mDisplayName.setText(patient.getName().getNameString());
+            } else if(null != patient.getDisplay()){
+                /*
+                if name is null, then we can get the name from 'display' which contains the ID and name
+                separated by a hyphen( - ).
+                 */
+                String patientName = patient.getDisplay().split("-")[1];
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/lastviewedpatients/LastViewedPatientRecyclerViewAdapter.java]: Make sure to remove trailing whitespaces.

**Comment by vansha10** [comment]
> Thanks for the PR @vansha10 . Please also align the name and id TextViews so the **right ends** match, right now the name and id position doesn't look good visually.

@f4ww4z The name and id right ends look aligned to me. Do you mean the name and download button?

**Comment by vansha10** [comment]
Aligned name and download button.
Screenshot_20190703-193229

**Comment by f4ww4z** [comment]
> @f4ww4z The name and id right ends look aligned to me. Do you mean the name and download button?

Yes sorry, I do mean the name and download button.

Looks good @vansha10 !

**Pull request merged**